iris-mpc-store/src/lib.rs

```diff
@@ -69,6 +69,7 @@ impl StoredIris {
 
 #[derive(Clone)]
 pub struct StoredIrisRef<'a> {
+    pub id:         usize,
```

Comment:
should this directly be an i64?

Reply:
yeah I guess this should represent it as what it actually is in pg, updated.